mountd/smbacl.c

```diff
@@ -135,17 +134,17 @@ static void smb_sid_to_string(char *domain, struct smb_sid *sid)
 
 	strncpy(domain, "S-", 2);
 	domain_len += 2;
-	len = g_sprintf(str, "%i", (int)sid->revision);
+	len = sprintf(str, "%i", (int)sid->revision);
```

Comment:
But you replaced strncpy with g_sprintf in this https://github.com/cifsd-team/ksmbd-tools/pull/247/commits/dfbebe6e62f3dbae25f783a1aa5e32e49a393e73

Reply:
yeah. sprintf is good enough. the g_ versions of normal *f functions offer no benefit over the C ones.

Reply:
@neheb, you replaced printf() with g_print(). Does it offer any benefit?

Reply:
I don't think so. One reason to get rid of it is that it's located in a different header.

Reply:
only g_sprintf() is needed for <glib/gprintf.h> ? other gprintf APIs can be used with glib.h ?

Reply:
The man page says this:

```
Note that the functions g_printf(), g_fprintf(), g_sprintf(), g_snprintf(), g_vprintf(), g_vfprintf(), g_vsprintf() and g_vsnprintf() are declared in the header gprintf.h which is not included in glib.h (otherwise using glib.h would drag in stdio.h), so you'll have to explicitly include <glib/gprintf.h> in order to use the GLib printf() functions. 
```